pages.fr/common/python.md

```diff
@@ -0,0 +1,28 @@
+# python
+
+> Interprète du langage Python.
+> Plus d'informations : <https://www.python.org>.
+
+- Démarre une invite de commande Python (REPL) :
+
+`python`
+
+- Exécute le script contenu dans un fichier Python :
+
+`python {{script.py}}`
```

Comment:
Could the values in `{{token syntax}}` be translated too?

Also, if you're translating a batch of pages, there's no need to create separate PRs for each one. You can lump them together into 1 big PR if you like - it makes the review process more efficient :D